src/components/AdminDashboard/AdminDashboard.jsx

```diff
@@ -0,0 +1,90 @@
+
+import './AdminDashboard.module.css';
+
+import { useBackend } from '../../contexts/BackendContext';
+import { useEffect, useState } from 'react';
+import { Box } from '@chakra-ui/react';
+
+
+const BusinessDashboard = () => {
+
+    // Created the use states
+  const { backend } = useBackend();
+  const [donationData, setDonationData] = useState([]);
+  const [businessDictionary, setBusinessDictionary] = useState([]);
+
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        // fetches the business table to be used in pending &total #
+        const businessResponse = await backend.get('/business');
+        setBusinessDictionary(businessResponse.data);
+
+        // fetches donation table data
+        const donationResponse = await backend.get('/donation/');
+        setDonationData(donationResponse.data);
+
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'backend'. Either include it or remove the dependency array.